test/conformance/helpers/tracing_test_helper.go

```diff
@@ -105,9 +104,9 @@ func tracingTest(
 func getTraceIDHeader(t *testing.T, evInfos []recordevents.EventInfo) string {
 	for i := range evInfos {
 		if nil != evInfos[i].HTTPHeaders {
-			sc := trace.RemoteSpanContextFromContext(propagation.TraceContext{}.Extract(context.TODO(), http.Header(evInfos[i].HTTPHeaders)))
+			sc := trace.SpanContextFromContext(propagation.TraceContext{}.Extract(context.TODO(), propagation.HeaderCarrier(evInfos[i].HTTPHeaders)))
```

Comment:
From the otel CHANGELOG:
> The concept of a remote and local Span stored in a context is unified to just the current Span. Because of this "go.opentelemetry.io/otel/trace".RemoteSpanContextFromContext is removed as it is no longer needed. Instead, "go.opentelemetry.io/otel/trace".SpanContextFromContex can be used to return the current Span. If needed, that Span's SpanContext.IsRemote() can then be used to determine if it is remote or not.